source/contracts/reporting/Universe.sol

```diff
@@ -56,6 +57,8 @@ contract Universe is DelegationTarget, ITyped, Initializable, IUniverse {
         require(isContainerForMarket(IMarket(msg.sender)));
         forkingMarket = IMarket(msg.sender);
         forkEndTime = block.timestamp + Reporting.forkDurationSeconds();
+        // We pre calculate the amount of REP needed to determine a winner in the fork REP migration contest
+        forkReputationGoal = reputationToken.totalSupply() / Reporting.forkRepMigrationVictoryDivisor();
```

Comment:
This opens up an attack that Id on't have a good solution for.

1. Attacker collects 99% of all REP.
2. Attacker triggers a fork.
3. Attacker does not migrate any REP to the fork (sacrifices bonds).
4. On new universe, attacker triggers another fork.
5. In the same block as the second fork is triggered, the attacker migrates 5% of their REP from original universe to intermediate universe and then from intermediate universe to final universe.
6. Attacker migrates remaining REP to truth universe.

This causes the fork to finalize the way the attacker wants (via early finalization at 51%, but only costing them dispute bonds (for first fork) + 5% of REP (for second fork).

Even if we didn't have the 51% auto-win rule for forks we would still have the same problem as the attacker could just hold their 94% of REP (minus bonds) until after the fork finalized.

One way we could tentatively solve this is by summing the total supply of reputation in the forking universe plus all of its ancestors.  The problem here is that technically that is an unbounded operation.  Perhaps its OK as the rate of growth will be really slow and we can address it later if that becomes a problem?

Reply:
Isn't the system already compromised in general if a malicious party acquires 99% of all REP?

While I'm sure this is a legit vulnerability we were previously hardcoding `11 * 10 ** 6 * 10 ** 18` for this number. After even one fork that number is arbitrary and results in a completely broken forking system.

Reply:
No, acquiring all REP is not considered unhealthy/unsafe.  It is only a problem if attacking the system requires you to _burn_ that REP.  The security model is built around the cost of an attack, not the up-front capital required to launch the attack.  The attack I described above is such that you need to acquire 99% of all REP, but only burn 5% of that REP.  You can then sell the remaining 94% back to the market to recoup most of your initial costs.